inconspiquous/gates/core.py

```diff
@@ -60,14 +69,14 @@ def pauli_prop(
             pauli_type: Either "X" or "Z" indicating the type of Pauli gate
 
         Returns:
-            A tuple of (X, Z) pairs for each output qubit, where True indicates
-            that the corresponding Pauli component should be applied to that output.
+            A PauliProp object where the `x` and `z` component determine whether
+            the corresponding Pauli component should be applied to that output.
 
         For example, for Hadamard gate:
-            - X propagates to Z: pauli_prop(0, "X") returns ((False, True),)
-            - Z propagates to X: pauli_prop(0, "Z") returns ((True, False),)
+            - X propagates to Z: pauli_prop(0, "X") returns ((x: False, z: True),)
+            - Z propagates to X: pauli_prop(0, "Z") returns ((x: True, z: False),)
         """
-        pass
+        ...
```

Comment:
just FYI you don't need this if you have a docstring

Reply:
```suggestion
```